src/turn.ts

```diff
@@ -32,14 +32,17 @@ export const requestTwilioTurnServer = async (accountSid: string, authToken: str
     const twilioIceServers = JSON.parse(JSON.stringify(token.iceServers));
     const iceServers: RTCIceServer[] = [];
     twilioIceServers.forEach((twilioIceServer: ITwilioIceServer) => {

```

Comment:
nit: this is a good fit for [`.map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) - not sure if you are familiar. 

Not blocking on this, more just commenting for future awareness 😸